### PR TITLE
jabber: evilify the key bindings

### PR DESCRIPTION
**Pull request by mssun**
Make jabber evil.

### CODE REVIEW

File: layers/+chat/jabber/packages.el [hunk 1]
```diff
@@ -15,16 +15,20 @@
...
-              "e" 'jabber-roster-edit-action-at-point
-              "g" 'jabber-display-roster
-              "i" 'jabber-get-disco-items
-              "j" 'jabber-muc-join
-              "q" 'bury-buffer
-              "r" 'jabber-roster-toggle-offline-display
-              "s" 'jabber-send-subscription-request
-              "v" 'jabber-get-version
-              "RET" 'jabber-roster-ret-action-at-point)))
+    :config (progn
```
Comment (StreakyCobra) [inline layers/+chat/jabber/packages.el]: The `progn` would better start on the next line to follow the style of the rest of the code :-)

Comment (mssun) [inline layers/+chat/jabber/packages.el]: Done.

**Comment by StreakyCobra** [comment]
And if you know how to squash commits you can do it :-) If not, don't worry we will deal with this.

**Comment by syl20bnr** [comment]
Thank you ! :+1: 
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**